.eslintrc

```diff
@@ -1,20 +1,28 @@
 {
-  extends: 'airbnb',
-  parserOptions: {
-    ecmaVersion: 7,
-    sourceType: 'module',
-    ecmaFeatures: {
-      jsx: true,
-      impliedStrict: true,
-      experimentalObjectRestSpread: true
-    }
-  },
+  extends: [
+    'airbnb',
+    'plugin:flowtype/recommended',
+  ],
+  plugins: ['flowtype'],
+  parser: 'babel-eslint',
   rules: {
     "import/no-unresolved": 0,
     "import/no-extraneous-dependencies": 0,
     "react/jsx-filename-extension": 0,
     "react/no-unused-prop-types": 0,
-    "no-console": [1, {allow: ["warn", "error"] }],
+    "no-console": [1, { allow: ["warn", "error"] }],
+
+    "flowtype/delimiter-dangle": [2, "always-multiline"],
+    "flowtype/no-dupe-keys": 2,
+    "flowtype/no-primitive-constructor-types": 2,
+    "flowtype/no-weak-types": 1,
+    "flowtype/object-type-delimiter": [2, "comma"],
+    "flowtype/semi": 2,
+  },
+  settings: {
+    flowtype: {
+      onlyFilesWithFlowAnnotation: true,
+    }
```

Comment:
It overrides the setting from recommended config: https://github.com/gajus/eslint-plugin-flowtype/blob/master/src/configs/recommended.json